security.md

```diff
@@ -0,0 +1,20 @@
+# Security policy
+
+We believe in transparency to mitigate security risks. All known vulnerabilities are available on 
+our [security page](docs/modules/ROOT/pages/security.adoc).
+
+We disclose such security issues only once a released version addressing the issue is available.
+
+# Reporting security vulnerabilities
+
+To ensure safety of our users, security process needs to happen privately.
+
+Here are the steps:
+
+ - 1. Reporter email the issues privately to `openpaas-james[AT]linagora.com`.
```

Comment:
Duplicated from security.adoc

Reply:
This was intendeed. The afoc antora file is "the souce of truth" but I'm not expecting people to find it. Hence I wanted to have a link and critical disclosure information there too, at the root of the repo.